File: perl/list/qw/src/list/list.pl
#!/usr/bin/perl

# qwによるリストで変数を初期化.
($x, $y, $z) = qw/10 20 30/; # qwでスペース区切りで変数を初期化.
print '$x = '."$x".', $y = '."$y".', $z = '."$z\n"; # $x, $y, $zを出力.

# qwによるリストで配列を初期化.
@array = qw/100 200 300/; # qwでスペース区切りで配列を初期化.
for ($i = 0; $i < 3; $i++){ # 3回繰り返す.
  print '$array['.$i.'] = '."$array[$i]\n"; # $array[$i]を出力.
}

File: python/list/multi_dimensional_list/src/list/list.py
#!/usr/bin/python
# coding: utf-8

# 要素が[[11, 12, 13], [21, 22, 23], [31, 32, 33]]の多次元リスト.
lst1 = [[11, 12, 13], [21, 22, 23], [31, 32, 33]];

# lst1[0]の出力.
print lst1[0]

# lst1[2][1]の出力.
print lst1[2][1] # 32が出力される.

# for-inで全要素を参照.
print "-----"
for sublst in lst1:
  for e in sublst:
    print e
